lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -332,48 +311,39 @@ defmodule RealtimeWeb.RealtimeChannel do
     {:stop, :shutdown, socket}
   end
 
+  def handle_info(:sync_presence = msg, socket) do
+    PresenceHandler.call(msg, socket)
```

Comment:
Nitpick: rename the function from `call` to something else, such as "handle" or "run". The reason is because `call` in Elixir is typically used for GenServers and synchronous communication, so I was originally thinking that this would cross a process (which would be a no-go as that would cause the whole socket to be copied :D).